gallery/src/styles/ColorPalette/ColorPalettePage.tsx

```diff
@@ -22,6 +23,9 @@ const ColorPalettePage = () =>
         CSS custom properties declared within the RSC base styles. The swatches below show all of the colors
         available in RSC along with their hex value and corresponding RSC variable.
       </p>
+      <NxWarningAlert>
+        The lime swatch is deprecated and is currently an alias for the green swatch.
```

Comment:
```suggestion
        The lime swatches are deprecated and are currently an alias for the green swatches.
```

Reply:
The semantics I've settled on, and which are reflected to varying degrees in the docs I've written in a variety of places, are that a single color value is a "sample", a group of all lightness variations of a single hue/saturation is a "swatch", and all the swatches together are a "palette".

Reply:
> a group of all lightness variations of a single hue/saturation is a "swatch", and all the swatches together are a "palette"

What's the difference between these two things?

Reply:
The definitions I've been using in my head:

Sample: 
![image](https://user-images.githubusercontent.com/3630091/127549734-59a886e3-8602-4874-aaea-9f9bf2a2b6c1.png)

Swatch:
![image](https://user-images.githubusercontent.com/3630091/127549788-59d0a083-27e8-451a-a97b-d840f3df58c7.png)

Palette:
![image](https://user-images.githubusercontent.com/3630091/127549988-201a5a8e-b0ca-4bc5-ae24-c8eb6b7896f7.png)

Reply:
If we want to say "swatch" means a single color (what I've been calling "sample"), then we need to decide what to call the thing I'm currently calling a "swatch".

Reply:
A swatch is by definition a single colour (technically a single sample since the same term is used for fabrics).

Based off what you have above: sample => swatch, swatch => palette, palette => palettes, because a palette can be all the blues and also all the colours.

Reply:
> Based off what you have above: sample => swatch, swatch => palette, palette => palettes, because a palette can be all the blues and also all the colours.

b0238cb7.  I've created RSC-646 to make sure the rest of the docs follow this meaning of the terms as well.